src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -103,33 +103,38 @@ struct ComputeNormalDotFluxes {
  * \brief Computes the penalty flux
  *
  * The penalty flux is given by:
+ *
  * \f{align}
  * F^*(\Psi) =& F(\Psi) = 0 \\
- * F^*(\Pi) =& F(\Pi)_{\mathrm{int}} + \\
- *         \frac{1}{2} p \left(U^-_{\mathrm{ext}} - U^-_{\mathrm{int}}\right) \\
- * F^*(\Phi_i) =& F(\Phi_i)_{\mathrm{int}} - \\
- *         \frac{1}{2} p \left(U^-_{\mathrm{ext}} - U^-_{\mathrm{int}}\right)
- * \f}
- * where \f$F^*\f$ is the normal dotted with the numerical flux and \f$F\f$ is
- * the normal dotted with the flux, which is computed in
- * ScalarWave::ComputeNormalDotFluxes.
+ * F^*(\Pi) =& F(\Pi)_{\mathrm{int}} +
+ *         \frac{1}{2} p \left(v^{\hat -}_{\mathrm{ext}} - v^{\hat
+ * -}_{\mathrm{int}}\right) \\ F^*(\Phi_i) =& F(\Phi_i)_{\mathrm{int}} -
+ *         \frac{1}{2} p \left(v^{\hat -}_{\mathrm{ext}} - v^{\hat
+ * -}_{\mathrm{int}}\right) \f}
+ *
+ * where \f$F^*\f$ is the interface normal dotted with numerical flux, \f$F\f$
+ * is the normal dotted with flux (computed in
+ * ScalarWave::ComputeNormalDotFluxes), and \f$v^{\hat -}\f$ is an incoming
+ * characteristic fields of the system (see
+ * characteristic_fields() for its definition). The penalty factor
+ * `p` is chosen to be unity.
  */
 template <size_t Dim>
 struct PenaltyFlux {
  private:
-  struct NormalTimesUPlus {
+  struct NormalTimesVPlus {
     using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
-    static std::string name() noexcept { return "NormalTimesUPlus"; }
+    static std::string name() noexcept { return "NormalTimesVPlus"; }
```

Comment:
I think these names are defaulted now. Would you mind trying to remove them?

Reply:
Are they defaulted even if the tag is not inheriting from `db::SimpleTag` (which these tags are not)?

Reply:
Oh, sorry. I thought these were option tags. Never mind!